packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -316,6 +316,9 @@ export abstract class PersistentStream<
 
     this.cancelIdleCheck();
 
+    // Ensure we don't leave a pending backoff operation queued.
+    this.backoff.cancel();
```

Comment:
It's a little weird that cancelling the idle and backoff timers look different. Is there any way to make these uniform? I suppose this happens because the backoff timer is held within the backoff object but the idle timer is held directly?

I think this may be non-actionable, but mentioning it anyway.

Reply:
acknowledged.